lib/ansible/plugins/action/__init__.py

```diff
@@ -803,7 +804,7 @@ def _remote_checksum(self, path, all_vars, follow=False):
         3 = its a directory, not a file
         4 = stat module failed, likely due to not finding python
         5 = appropriate json module not found
-        '''
+        """
```

Comment:
Add a runtime deprecation

Reply:
Ok. I was wondering how we actually handle that. Do we just do a `display.deprecate()` since it's an internal call and the user might not know what to do with that information?

Reply:
yep, but mostly a note that 'the plugin author should update his code'